src/Tickets/Admin/All_Tickets/List_Table.php

```diff
@@ -241,18 +522,35 @@ public function prepare_items() {
 		 *
 		 * @return array
 		 */
-		$args = apply_filters( 'tec_tickets_all_tickets_table_query_args', $args );
+		return apply_filters( 'tec_tickets_all_tickets_table_query_args', $args );
+	}
 
-		$items = [];
+	/**
+	 * Prepares the list of items for displaying.
+	 *
+	 * @since TBD
+	 */
+	public function prepare_items() {
+		global $wpdb;
+
+		$args               = $this->get_query_args();
+		$tickets_repository = tribe_tickets()->by_args( $args );
+		$total_items        = $tickets_repository->found();
+		$items              = $tickets_repository->all();
+		foreach ( $items as $i => $item ) {
+			$this->items[] = Tribe__Tickets__Tickets::load_ticket_object( $item->ID );
+		}
 
-		$pagination_args['total_items'] = count( $items );
+		$pagination_args = [
+			'total_items' => $total_items,
+			'per_page'    => $this->get_items_per_page( $this->per_page_option ),
+		];
 
 		$this->_column_headers = [
 			$this->get_table_columns(),
 			$this->get_hidden_columns(),
 			$this->get_sortable_columns(),
 		];
-		$this->items           = $items;
```

Comment:
Dont we need $this->items defined ?

Reply:
It's defined in the [parent class](https://developer.wordpress.org/reference/classes/wp_list_table/#properties)